src/core_plugins/metrics/public/less/misc.less

```diff
@@ -84,11 +84,16 @@
   flex-direction: column;
   flex: 1 0 auto;
   padding: 10px;
-  position: absolute;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
+
+  // The following is commented out for posterity to ensure no ill-effects
+  // But the absolute positioning broke reports and was fighting with display: flex,
+  // Setting the max-width to 100% fixed the issue abs positioning was trying to
+  // position: absolute;
```

Comment:
Personally, I'm not a fan of leaving code in comments. I'd just remove it, and anyone who's interested can look at git blame / history to understand things here. I like the idea of commenting why you need a max-width, though, as that wouldn't be apparent at all!

Reply:
👍